Gordon360/Controllers/ProfilesController.cs

```diff
@@ -611,5 +629,20 @@ public ActionResult<IEnumerable<MembershipHistoryViewModel>> GetMembershipHistor
 
             return Ok(membershipHistories);
         }
+
+        /// <summary>
+        /// Return a list of mail destinations' descriptions.
+        /// </summary>
+        /// <returns> All Mail Destinations</returns>
+        [HttpGet]
+        [Route("mail_stops")]
+        public ActionResult<IEnumerable<string>> GetMailStops()
+        {
+            var involvements = _webSQLContext.Mailstops.Select(m => m.code)
+                                   .Distinct()
+                                   .Where(d => d != null)
```

Comment:
```suggestion
                                   .Distinct()
                                   .Where(d => d != null)
```
webSQL.dbo.Mailstops is a lookup table. Thus, the code will always be unique and not null.